Makefile

```diff
@@ -115,10 +118,12 @@ test: manifests generate fmt vet envtest ## Run tests.
 .PHONY: build
 build: generate fmt vet ## Build manager binary.
 	go build -o bin/manager main.go
+	go build -o $(ISTIO_INSTALL_BIN_PATH) cmd/istio-install/main.go
 
 .PHONY: run
 run: manifests generate fmt vet ## Run a controller from your host.
-	go run ./main.go
+	go build -o $(ISTIO_INSTALL_BIN_PATH) cmd/istio-install/main.go
```

Comment:
this line duplicates with 121, could we eventually move istio-install building in separate task?

Reply:
Thanks for pointing it out.

Reply:
Changed